index.js

```diff
@@ -138,11 +143,7 @@ exports.init = function (sbot, config) {
             }
             else {
               log('tunnel:connect - portal connected, tunnel to target:'+opts.target)
-              cb(null, rpc.tunnel.connect({target: opts.target, port: opts.port}, function (err) {
-                if(err)
-                  log('tunnel:connect - failed to connect to target:'+opts.target+' '+err.message)
-                //how to handle this error?
-              }))
+              cb(null, rpc.tunnel.connect({target: opts.target, port: opts.port}))
```

Comment:
tunnel.connect does not need a callback, because it synchronously returns, so this inner callback was never being called.